### PR TITLE
tail: Add test for `tail -f`.

### PR DESCRIPTION
**Pull request by ProgVal**

### CODE REVIEW

**Comment by Heather** [comment]
@ProgVal breaks travis seems like

**Comment by ProgVal** [comment]
That's a bug that happens only in the stable version of Rust. You can reproduce it here: https://play.rust-lang.org/?gist=2a9c627a95b45c1fff8ddc03b367a0cc&version=beta&backtrace=0

Expected behavior: prints `barbaz`.
In stable: prints `bar`.

**Comment by ProgVal** [comment]
So… I'm not sure what you want to do about this.
Skip the test on Rust stable?

(PS: I rebased my commits on the current version of your master branch)

**Comment by Heather** [comment]
@ProgVal not sure but there should be some solution to not break travis, or maybe change travis to allow breakage for stable temporary /cc @ebfe @jbcrail

**Comment by ebfe** [comment]
Now that we finally got everything working on stable I think we should keep it that way. Skipping the test is not an option if it hides a bug on stable, IMO.

**Comment by ebfe** [comment]
Also since the test in question worked fine before, i suspect the cause is actually the changes in UCommand::run().

**Comment by ProgVal** [comment]
No, that's a test I added. The issue is with appending to a test file.

**Comment by ebfe** [comment]
> No, that's a test I added. The issue is with appending to a test file.
Indeed, I misread the travis log as the previous test timing out.

**Comment by ProgVal** [comment]
Reported to rustc: https://github.com/rust-lang/rust/issues/32801

**Comment by Heather** [comment]
@ProgVal merging this will break other PRs so either it should be commented with `TODO` or will be merged when upstream bug will go I guess...

**Comment by ProgVal** [comment]
Fixed. One build failed with Rust Nightly for some unrelated reason

File: Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ BUILDDIR      := $(BASEDIR)/target/${PROFILE}/
 PKG_BUILDDIR  := $(BUILDDIR)/deps/
 
 BUSYBOX_ROOT := $(BASEDIR)/tmp/
-BUSYBOX_VER := 1.24.1
+BUSYBOX_VER := 1.22.0
```
Comment (ebfe) [inline Makefile]: Unrelated change

Comment (Heather) [inline Makefile]: @ebfe ahm... I've just merged without seen this, please revert only this change

Comment (ProgVal) [inline Makefile]: oops, sorry
